erts/emulator/beam/erl_db.c

```diff
@@ -3425,9 +3425,9 @@ static void fix_table_locked(Process* p, DbTable* tb)
     erts_refc_inc(&tb->common.ref,1);
     fix = tb->common.fixations;
     if (fix == NULL) { 
-	get_now(&(tb->common.megasec),
-		&(tb->common.sec), 
-		&(tb->common.microsec));
+	get_sys_now(&(tb->common.megasec),
+		    &(tb->common.sec), 
+		    &(tb->common.microsec));
```

Comment:
Oh right, it's the diff that is wrong because leading tabs misaligns code by one. Nevermind. These mixed tabs and spaces still are a PITA though. :(